client/my-sites/plans/jetpack-plans/jetpack-complete-page/components/cta-buttons/style.scss

```diff
@@ -0,0 +1,40 @@
+.jetpack-complete-page__cta-buttons {
+	display: inline-flex;
+	align-items: center;
+	justify-content: center;
+	width: 100%;
+	& > * {
+		flex: 1;
+	}
```

Comment:
Design mock-ups don't seem to indicate that buttons should grow, which is what this rule does. (Also, it's quite vague. If we kept it, I'd recommend simply `.button` instead of `> *`; but that's neither here or there.)

Instead, let's remove the `flex` rule here entirely.